code/game/objects/items/documents.dm

```diff
@@ -115,3 +128,90 @@
 		. += "<span class='notice'>Wait a minute, this isn't an encrypted inspection report! You should throw it away.</span>"
 	else
 		. += "<span class='notice'>Wait a minute, this thing's blank! You should throw it away.</span>"
+
+/**
+ * # Fake N-spect scanner
+ *
+ * A clown variant of the N-spect scanner
+ *
+ * This prints fake reports with garbage in them,
+ * can be set to print them instantly with a screwdriver.
+ * By default it plays the old "woody" scanning sound, but it can be set
+ * to play the normal N-spect scanner sound with a multitool
+ */
+/obj/item/inspector/clown
+	print_sound = 'sound/items/biddledeep.ogg'
+
+/obj/item/inspector/clown/attack(mob/living/M, mob/living/user)
+	. = ..()
+	print_report()
+
+/obj/item/inspector/clown/print_report()
+	// Create our report
+	var/obj/item/paper/fake_report/slip = new(get_turf(src))
+	slip.generate_report(get_area(src))
+	playsound(src, print_sound, 50, FALSE)
+
+/obj/item/inspector/clown/attackby(obj/item/I, mob/user, params)
+	if(I.tool_behaviour == TOOL_MULTITOOL)
+		if(print_sound == 'sound/items/biddledeep.ogg')
+			print_sound = 'sound/machines/high_tech_confirm.ogg'
+			to_chat(user, "<span class='notice'>You set the device's bleep setting to normal mode")
+		else if(print_sound == 'sound/machines/high_tech_confirm.ogg')
+			print_sound = 'sound/items/bikehorn.ogg'
+			to_chat(user, "<span class='notice'>You set the device's bleep setting to honk mode")
+		else
+			print_sound = 'sound/items/biddledeep.ogg'
+			to_chat(user, "<span class='notice'>You set the device's bleep setting to classic mode")
+	else if(I.tool_behaviour == TOOL_SCREWDRIVER)
+		if(print_time == 0)
+			print_time = 5 SECONDS
+			to_chat(user, "<span class='notice'>You set the device's scanning speed to SLOW.")
+		else
+			print_time = 0
```

Comment:
A 0 print time means they can just spam the floors with shit that no one cares about.

I realise this is your intent, and I strongly disagree.

Reply:
New janitor content

Reply:
clowns can already spam floors with bananas, which is arguably way worse than this
make this cost bananium or something as a result if it doesn't already

Reply:
Nah, we really need to have at least a short delay on this. We had another item that people could spam paper with and it caused a fair bit of lag.

Reply:
Changed.